pkg/logger/logger.go

```diff
@@ -80,9 +75,9 @@ func SetupErrorLogger() {
 // getLogFilePath retuns the log file path depending on the OS.
 func getLogFilePath() string {
 	if runtime.GOOS == "windows" {
-		return WindowsLogFilePath
+		return WindowsInstallPath + "logs\\aws-otel-collector.log"
```

Comment:
Looks like the "Logs" changed to "logs", it is intended?

Reply:
No, thanks for catching it.